### PR TITLE
WIP - Add support for v1/issuer_fraud_records endpoint

### PR DESCRIPTION
**Pull request by fay-stripe**

### CODE REVIEW

File: src/Stripe.Tests.XUnit/issuer_fraud_records/when_listing_issuer_fraud_records.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+using System.Collections.Generic;
+using System;
+using System.Linq;
+using Xunit;
+
+namespace Stripe.Tests.XUnit
+{
+
+
+    public class when_listing_issuer_fraud_records
```
Comment (brandur-stripe) [inline src/Stripe.Tests.XUnit/issuer_fraud_records/when_listing_issuer_fraud_records.cs]: Can you make sure to just tighten up the spacing in these for the final version?

File: src/Stripe.net/Services/IssuerFraudRecords/StripeIssuerFraudRecordListOptions.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+namespace Stripe
+{
+    public class StripeIssuerFraudRecordListOptions : StripeListOptions
+    {
+        [JsonProperty("charge")]
+        public string ChargeId { get; set; }
+
+        [JsonProperty("date")]
+        public StripeDateFilter Date { get; set; }
```
[the remark this answers is not in the document]
Comment (fay-stripe) [reply src/Stripe.net/Services/IssuerFraudRecords/StripeIssuerFraudRecordListOptions.cs]: I thought all the List methods take by default: https://stripe.com/docs/api#list_issuer_fraud_records (ending_before, starting_after)? 
https://git.corp.stripe.com/stripe-internal/pay-server/blob/master/lib/disputes/issuer_fraud_record/api/all_method.rb#L54

File: src/Stripe.Tests.XUnit/issuer_fraud_records/when_listing_issuer_fraud_records.cs [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+        StripeList<StripeIssuerFraudRecord> result;
+
+        public when_listing_issuer_fraud_records()
+        {
+
+            Cache.GetSubscription();
+            Cache.GetSubscription();
+            Cache.GetSubscription();
+            Cache.GetSubscription();
```
[the remark this answers is not in the document]
Comment (fay-stripe) [reply src/Stripe.Tests.XUnit/issuer_fraud_records/when_listing_issuer_fraud_records.cs]: Oops, those were c/p from another test. Def not needed

File: src/Stripe.Tests.XUnit/issuer_fraud_records/when_listing_issuer_fraud_records.cs (hunk 1 above)
Comment (fay-stripe) [reply src/Stripe.Tests.XUnit/issuer_fraud_records/when_listing_issuer_fraud_records.cs]: Oh I copied the spacing from when_listing_invoices, sorry I thought that was convention. Will remove spaces

File: src/Stripe.net/Entities/StripeIssuerFraudRecords.cs [hunk 4]
```diff
@@ -0,0 +1,43 @@
+using System;
```
[the remark this answers is not in the document]
Comment (fay-stripe) [reply src/Stripe.net/Entities/StripeIssuerFraudRecords.cs]: Done!

**Comment by fay-stripe** [comment]
@brandur-stripe I'm not sure what you mean by separating out the specs for List and Retrieve - I checked the files for `CountrySpec`, and I assume you mean `CountrySpecService`? But it also has the `list()` and `get()` method in the same file.

**Comment by brandur-stripe** [comment]
>  I'm not sure what you mean by separating out the specs for List and Retrieve - I checked the files for CountrySpec, and I assume you mean CountrySpecService? But it also has the list() and get() method in the same file.

Ah, I meant for the specs, not for the other files. You have a `when_listing_issuer_fraud_records`, and may want a similar file for retrieval as well.

**Comment by remi-stripe** [comment]
I'm going to soft-close this one for now. We can always reopen it in the future if we end up shipping this resource more broadly.

**Pull request closed without merging**